types/immutability-helper/index.d.ts

```diff
@@ -1,6 +1,7 @@
 // Type definitions for immutability-helper v2.0.0
 // Project: https://github.com/kolodny/immutability-helper
 // Definitions by: Sean Kelley <https://github.com/seansfkelley>
+// Definitions by: Adam Gordon <https://github.com/xmrwhite>
```

Comment:
Add your name to the end of the list: `// Definitions by: Sean Kelley <https://github.com/seansfkelley>, Adam Gordon <https://github.com/xmrwhite>`